docker/README.md

```diff
@@ -1,5 +1,9 @@
 # Run The OpenZiti Tunneler with Docker
 
+:::important
```

Comment:
this is "docusaurus" syntax.  a while back I added processing to docusaurus to handle "github flavored syntax"

Change this to:
```

>[!NOTE]
> Persistent, writeable volumes are ....
```

and it'll render as a note in docusaurus and properly in github

Reply:
or WARNING or CAUTION

![image](https://github.com/user-attachments/assets/694ed1f8-266f-4167-889a-c2e5d78e3220)

![image](https://github.com/user-attachments/assets/d3dedc8a-0275-4a79-9e1a-050f6b41a658)